qmd/sets.qmd

```diff
@@ -15,12 +15,28 @@ Sets are lists of categories and other parts. For example, `Collection category
 
 Sets can have parts that are reused. For example, there are different quality measures for sample collection (e.g. `Sample not collected`), PCR measures (`Inhibition preset`), and sequencing measures (`Low depth of coverage`), but all quality sets share the `No quality concerns` indicator.
 
-```{r, echo=FALSE, warning=FALSE}
+```{r, echo=FALSE, warning=TRUE}
 # Utility Functions
 # Source utility functions
 source("R/qmd-utils.R")
 # Source table formatting
 source("R/format-table.R")
+# Source warning utils
+source("R/warning-utils.R")
+source("R/template-utils.R")
+
+# Display string templates
+header_template <-
+  '## {partLabel} {{ #{partID} }}
+  <a href=\"/parts.html#{partID}\">({partID})</a>. {partDesc} {if (partInstr %!=na% constants$dictionary_missing_value_replacement) partInstr else "" }'
```

Comment:
I don't think templates should have any logic in them, we should keep them clean with just the variables. This will also help with debugging.

It will also help with refactoring common pieces of coded, for example the code in this and the next template where you replace missing value with ""